src/cat-life-event.tsx

```diff
@@ -6,9 +6,40 @@ export type Props = {
 
 export default (): FC<Props> => {
   const component = (props: Props): ReactElement => {
-    return <>
-      {props.children}
-    </>
+    return <div>
+      <h2>Cat life events</h2>
+      <table>
+        <thead>
+          <tr>
+            <td>date</td>
+            <td>event</td>
+            <td>cat</td>
+            <td>lives left</td>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>2022.11.27</td>
+            <td>lost a life</td>
+            <td>Acylsm</td>
```

Comment:
Where does this name come from?

Reply:
``` js
const foo = bar()
```

Change `foo` to `bar`.

This is a better cat name:
```suggestion
            <td>Garfield</td>
```

Reply:
Before merging:

- [ ] Clean up typos
- [ ] Explain how cat names appeared